src/test/java/com/notably/logic/SuggestionTestUtil.java

```diff
@@ -0,0 +1,157 @@
+package com.notably.logic;
```

Comment:
I just thought about this more. Do you think it's better to put this in the `com.notably.logic` or `com.notably.logic.suggestion`?

Reply:
ok will move it to the latter!